analog-minimalist/usr/share/asteroid-launcher/watchfaces-img/asteroid-logo-black.svg

```diff
@@ -0,0 +1,247 @@
+     id="path1075" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 264.58371,296.50677 10.64059,10.64059 -10.64059,10.64058 -10.64059,-10.64058 z"
+     id="path1077" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 275.2243,285.86618 10.64058,10.64059 -10.64058,10.64059 -10.64059,-10.64059 z"
+     id="path1079" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 275.2243,285.86618 10.64058,10.64059 -10.64058,10.64059 -10.64059,-10.64059 z"
+     id="path1081" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 285.86488,275.2256 10.64059,10.64058 -10.64059,10.64059 -10.64058,-10.64059 z"
+     id="path1083" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 253.94312,264.58501 10.64059,10.64059 -10.64059,10.64058 -10.64059,-10.64058 z m 42.56235,0 10.64059,10.64059 -10.64059,10.64058 -10.64059,-10.64058 z"
+     id="path1085" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 243.30253,253.94441 10.64059,10.6406 -10.64059,10.64059 -10.64059,-10.64059 z m 21.28118,0 10.64059,10.6406 -10.64059,10.64059 -10.64059,-10.64059 z m 42.56235,0 10.64058,10.6406 -10.64058,10.64059 -10.64059,-10.64059 z"
+     id="path1087" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 296.50547,243.30383 10.64059,10.64058 -10.64059,10.6406 -10.64059,-10.6406 z"
+     id="path1089" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 264.58371,232.66324 10.64059,10.64059 -10.64059,10.64058 -10.64059,-10.64058 z"
+     id="path1091" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 285.86488,232.66324 10.64059,10.64059 -10.64059,10.64058 -10.64058,-10.64058 z"
+     id="path1093" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 285.86488,232.66324 10.64059,10.64059 -10.64059,10.64058 -10.64058,-10.64058 z"
+     id="path1095" /><path
+     style="opacity:1;fill:#ffffff;fill-opacity:1;stroke-width:0.433915"
+     d="m 275.2243,222.02265 10.64058,10.64059 -10.64058,10.64059 -10.64059,-10.64059 z"
+     id="path1097" /><image
+     width="470.59811"
+     height="174.20599"
+     preserveAspectRatio="none"
+     xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAYUAAACQCAYAAADnT3fUAAAAAXNSR0IArs4c6QAAH3pJREFUeF7t
+XQncvtWYvi4zNHZmhkKbFiQRlbShyV6WiJAYMqFkLGEsxURNY0uGQrINRbYi2RKSfS2hZCLLGPsY
+yxjbNb/r/3++z/f/+t73Oc9ynuV97vv3e3//+M65z32uc95zvWe575sICQQCgUAgEAgECgQYSAQC
+gUAgEAgEAksIBCnEXAgEAoFAIBBYRiBIISZDIBAIBAKBQJBCzIFAIBAIBAKBKyMQO4WYFYFAIBAI
+BAKxU4g50B4CknYB8Nn2NK7T9FKST2pZZ6gLBAKBEgTW3ClI2hzAo0aI3iUk31LFbkm7A7hblTot
+l/0RyZNa1tmpOkm2/3EtN/oTADcm+fuW9Sark7QnAH9CukHgZJK/WGoq8O8G9BWtrMN/Fin4i/Dx
```

Comment:
More than half the file size is to show "xatt" which doesn't really add much to the design, except perhaps for the author.  I'd recommend omitting it.